package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "JSONCrush",
```

Comment:
Too bad the author didn't claim the [`jsoncrush` package name](https://www.npmjs.com/package/jsoncrush), but overloading via different casing doesn't seem like a great solution here: this will cause problems on filesystems that aren't strictly case-sensitive, e.g. apple filesystem. Are there any known issues with npm installing a namespaced package from a git repo? If not, consider something like `@killedbyapixel/jsoncrush`.